worlds/dlcquest/Items.py

```diff
@@ -94,40 +97,65 @@ def create_trap_items(world, World_Options: Options.DLCQuestOptions, trap_needed
     return traps
 
 
-def create_items(world, World_Options: Options.DLCQuestOptions, locations_count: int, random: Random):
+def create_items(world, world_options: Options.DLCQuestOptions, locations_count: int, random: Random):
     created_items = []
-    if World_Options.campaign == Options.Campaign.option_basic or World_Options.campaign == Options.Campaign.option_both:
```

Comment:
You can remove the "Options." prefix using the new system and slightly different imports. It'll considerably reduce bloat in this whole file (and others, maybe?)

Reply:
I don't want to switch them all (maybe another time) and I don't want both system at the same time